CHANGELOG.md

```diff
@@ -31,6 +31,10 @@
   used.  
   [Marcelo Fabri](https://github.com/marcelofabri)
   [#2227](https://github.com/realm/SwiftLint/issues/2227)
+* Stop using `init(_builtinStringLiteral:utf8CodeUnitCount:isASCII:)` of 
```

Comment:
I don't think this warrants a changelog entry since there are no user-facing changes.

Reply:
Removed.